### ISSUE TITLE
Repo niet laten openen in container

### ISSUE THREAD

**Issue by ralphniels** [hanaim-webtech/wt-is-beroepsproduct-env#6]
Bij het importeren van de repo in VSC wordt de gebruiker gevraagd deze te openen in de container. Als je dit doet, kan dit tot wat kleine problemen leiden. Het lijkt daarentegen geen voordelen te hebben. Ik begreep van Jorg dat deze vraag uit te schakelen is in de configuratie, maar ik weet niet meer hoe... :-/

**Comment by fritzvd** [comment]
Dag Ralph, ik heb het niet getest, maar ik denk dat deze folder daarvoor weg zou moeten.

**Comment by jvisch** [comment]
Klopt, die map .devcontainer moet weg. Ik zal een pull request doen.
